### PR TITLE
Add UVIS support

### PR DESCRIPTION
**Pull request by rosteen**
Adds support for the G280 grism for UVIS. In-progress, creating draft PR for purpose of sharing work.

### CODE REVIEW

**Comment by rosteen** [comment]
Alright, I think this is ready for review. I'll want to get SME input on whether the accuracy is good enough, but [this calibration paper](https://www.stsci.edu/files/live/sites/www/files/home/hst/instrumentation/wfc3/documentation/instrument-science-reports-isrs/_documents/2017/WFC3-2017-20.pdf) says that the accuracy is +/-7 Angstrom (I assume where the trace is decent), which seems to match what I'm seeing in at least some CCD locations.

It would also be helpful @duytnguyendtn to have your help double checking that I didn't inadvertently break anything on the IR side with all of this. There will also likely be a little reconciling to do with your changes in the `astrogrism` repo.

**Review by duytnguyendtn — APPROVED** [review]
I copied over the unit tests introduced in https://github.com/spacetelescope/astrogrism/pull/3 and they all passed without much fuss. Unit tests are wonderful :) Good work with coding in a way to not break the IR side of things!

I ran through the `UVIS_testing` notebook, all of which ran nicely within my own tolerance of "acceptable" deviation. Everything seems technically sound! I'll mark my approval now, but not merge it as it seems you want to get the OK from our SMEs first.

**Comment by rosteen** [comment]
Merging, per some planning done offline. May follow up with enhancements/fixes later if needed for the forward transforms.

**Pull request merged**
